libs/http/include/http/server.hpp

```diff
@@ -331,6 +342,79 @@ class HTTPServer : public AbstractHTTPServer
     });
   }
 
+  void AddDefaultRootModule()
```

Comment:
My feeling is that this should be not be contained here in the HTTP server. I can we not move this to an external module and then provide the accessors for the routes via the HTTP server interface.

Reply:
Ok, possible but—Either way the server should be aware of this module. Even if the module's definition itself is contained in a separate file, the module is added via server's special method (well, unless we declare it server's friend — which still requires changes to server's code).